analyzer-java/src/main/java/com/infosupport/ldoc/analyzerj/descriptions/TypeTypeFilter.java

```diff
@@ -2,13 +2,27 @@
 
 import java.util.Objects;
 
+/**
+ * I don't know exactly what this is for ?.
+ */
 public class TypeTypeFilter {
 
+  /**
+   * Check if given object is a TypeType.CLASS ?.
+   *
+   * @param obj ?
+   * @return ?
+   */
   @Override
   public boolean equals(Object obj) {
     return obj instanceof TypeType && Objects.equals(TypeType.CLASS, obj);
   }
 
+  /**
+   * ?.
+   *
+   * @return int 0
```

Comment:
Currently this method only exists to satisfy sonarcloud. In essence this method returns an integer of the current instance of the class. By default we have it set to 0 as we dont use it.

Reply:
So what would be a good description then?

Reply:
Something like this might work:

    /** Returns some integer. TypeTypeFilter is not intended to be used as a hash key. */